### DISCUSSION TITLE
Distance between website's name and "ABOUT, WORK, CONTACT"

### DISCUSSION THREAD

**Discussion by bluefander** [kdevo/osprey-delight#16] [Q&A] [answered]
Hi

I have another question:

Where can I change the distance between the name of the website and where is written "ABOUT, WORK, CONTACT"?

Thanks
![distance](https://user-images.githubusercontent.com/85159577/120890463-15ed3200-c603-11eb-9f19-4e867d560b31.png)

**Comment by kdevo** [comment, marked as answer]
Hello,

The bottom-attached navbar (i.e. the way it's done now) is a crucial design element of the theme, so it would help to know what you are trying to achieve in particular.

You would need to adjust multiple lines:

1. The navbar position is determined by absolutely positioning it from the bottom (you can use relative values there, too):

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/sass/_nav.scss#L33-L35

2. The rest of the body starts after 100% of the viewport height, and thus the navbar would need to be adjusted in accordance with it (so that it still looks "snap fitted"):

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/sass/_layout.scss#L35

https://github.com/kdevo/osprey-delight/blob/9b1fea2b45dc73e9ee45b477ea3011cb879de510/assets/sass/_layout.scss#L56-L58

Tip: You can use the developer tools of your browser (usually accessible via F12) to inspect site elements and play around with the values.

**Reply by bluefander** [reply]
Where you have the picture "KDEVO" I'd like to put a somewhat bigger picture. And I wanted to increase the distance where I drawed the yellow arrow. Thanks to your hints I could adjust it to my liking!

**Reply by kdevo** [reply]
Understood. Cool that it worked for you! 

Could you please mark my answer as resolution by pushing the "tick icon"?
![image](https://user-images.githubusercontent.com/10863868/121085089-b9943900-c7e1-11eb-9433-55ab0f54b5af.png)

If you like the theme, please also consider giving it a star - helps with growing a user-base :smile:

**Reply by kdevo** [reply]
Also, before I forget: Your input in the "survey" regarding the feature selection for the theme's next iteration is welcome, see #12
